### PR TITLE
Fix broken link and typos in Hydra 5-min tutorial

### PR DESCRIPTION
**Pull request by ashutoshgngwr**

### CODE REVIEW

File: docs/hydra/5min-tutorial.md [hunk 1]
```diff
@@ -6,10 +6,10 @@ title: 5 Minute Tutorial
 To get started quickly, we provide a Docker Compose based example for setting up ORY Hydra, a PostgreSQL instance
 and an exemplary User Login & Consent App. You need to have the latest Docker as well as Docker Compose version installed.
 
-<img src="../../images/docs/hydra/oauth2-flow.gif" alt="OAuth2 Flow">
+<img src="../../website/static/images/docs/hydra/oauth2-flow.gif" alt="OAuth2 Flow">
```
Comment (aeneasr) [inline docs/hydra/5min-tutorial.md]: This is also wrong - the right URL is: `img src="/website/static/images/docs/hydra/oauth2-flow.gif"`

Comment (ashutoshgngwr) [reply docs/hydra/5min-tutorial.md]: Thanks for pointing out. Done!

File: docs/hydra/5min-tutorial.md [hunk 2]
```diff
@@ -6,10 +6,10 @@ title: 5 Minute Tutorial
 To get started quickly, we provide a Docker Compose based example for setting up ORY Hydra, a PostgreSQL instance
 and an exemplary User Login & Consent App. You need to have the latest Docker as well as Docker Compose version installed.
 
-<img src="../../images/docs/hydra/oauth2-flow.gif" alt="OAuth2 Flow">
+<img src="/website/static/images/docs/hydra/oauth2-flow.gif" alt="OAuth2 Flow">
```
Comment (aeneasr) [inline docs/hydra/5min-tutorial.md]: Almost! :D The right url is: `src="/images/docs/hydra/oauth2-flow.gif"`, see:

* https://www.ory.sh/images/docs/hydra/oauth2-flow.gif

Comment (ashutoshgngwr) [reply docs/hydra/5min-tutorial.md]: Hang on! Let me try jumping off a cliff first!

On a side note, I was a victim of markdown preview on GitHub. Something should be done about this.

Comment (aeneasr) [reply docs/hydra/5min-tutorial.md]: Oh I see - yeah the build pipeline is a bit different and doesn't really reflect the repository structure. I'm not sure if we can actually fix that :/

Comment (ashutoshgngwr) [reply docs/hydra/5min-tutorial.md]: @aeneasr How about now?

Comment (aeneasr) [reply docs/hydra/5min-tutorial.md]: 👍 thank you for updating this :)

Comment (ashutoshgngwr) [reply docs/hydra/5min-tutorial.md]: @aeneasr Least I can do!

**Pull request merged**